examples/bind-itempicker-list-jsx/src/app.jsx

```diff
@@ -0,0 +1,49 @@
+import { Color, contentView, Stack, TextView } from 'tabris';
+import { List } from 'tabris-decorators';
+import { ExampleComponent } from './ExampleComponent';
+import { generate, Person } from './Person';
+
+/** @type {List<Person>} */
+const items = List.from(generate(10));
+
+contentView.append(
+  <Stack stretch alignment='stretchX' padding={12} spacing={12}>
+    <ExampleComponent stretch background={Color.silver} persons={items}/>
+    <TextView markupEnabled font={{size: 24}} onTapLink={e => modify(e.url)}>
+      Modify List: <a href='set'>set item</a> <a href='modify'>modify item</a>
+      &nbsp;<a href='insert'>insert item</a> <a href='remove'>remove item</a>
```

Comment:
This non-breaking space was probably not intended?

Reply:
It is. In JSX...

```
<a>foo</a>
<a>bar</b>
```
 becomes

```
<a>foo</a><a>bar</b> 
```

rendered as


```
foobar
```

Reply:
It is. In JSX...

```
<a>foo</a>
<a>bar</b>
```
 becomes

```
<a>foo</a><a>bar</b> 
```

rendered as


```
foobar
```